util/src/org/commcare/util/cli/ApplicationHost.java

```diff
@@ -94,7 +106,19 @@ private boolean loopSession() throws IOException {
                 //TODO: Command language
                 if(input.startsWith(":")) {
                     if(input.equals(":exit") || input.equals(":quit")) {
+                        System.exit(-1);
+                    }
+                    if (input.startsWith(":update")) {
+                        mUpdatePending = true;
+
+                        if(input.contains("-f")) {
+                            mForceLatestUpdate = true;
+                        }
+                        return true;
+                    }
 
+                    if(input.equals(":home")) {
+                        return true;

```

Comment:
This method, ```loopSession```,  always returns ```true```. Is this expected? Could we just make it return void instead?